06-building-a-program-start-to-finish/README.md

```diff
@@ -0,0 +1,7 @@
+### What
```

Comment:
Love how you created a README for the context. Let's see how that helps the learner and consider doing the same for other contexts.